### PR TITLE
fix bug: close io stream in test

### LINKED ISSUE #69: unuseful files are left after running Junit test
two files:
./TsFileReadPageInMem
./src/test/resources/perTestInputData

### PR DESCRIPTION
**Pull request by liukun4515**
close #69

### CODE REVIEW

File: src/test/java/cn/edu/tsinghua/tsfile/timeseries/read/QueryEnginePerf.java [hunk 1]
```diff
@@ -193,6 +193,7 @@ static public void writeToFile(FileSchema schema) throws InterruptedException, I
 			innerWriter.write(record);
 			lineCount++;
 		}
+		in.close();
```
Comment (kr11) [inline src/test/java/cn/edu/tsinghua/tsfile/timeseries/read/QueryEnginePerf.java]: Considering time cost testing, it's better to move `in.close()` after the next line, `endTime ...`

File: src/test/java/cn/edu/tsinghua/tsfile/timeseries/read/ReadPerf.java [hunk 2]
```diff
@@ -212,6 +212,7 @@ static public void writeToFile(FileSchema schema) throws InterruptedException, I
             innerWriter.write(record);
             lineCount++;
         }
+        in.close();
```
Comment (kr11) [inline src/test/java/cn/edu/tsinghua/tsfile/timeseries/read/ReadPerf.java]: Considering time cost testing, it's better to move `in.close()` after the next line, `endTime ...`

File: src/test/java/cn/edu/tsinghua/tsfile/timeseries/write/ReadPageInMemTest.java [hunk 3]
```diff
@@ -227,6 +227,12 @@ public void MultiDeltaObjectTest() {
 		assertEquals(null, innerWriter.query("root.car.d1", "s5").get(1));
 		assertEquals(null, innerWriter.query("root.car.d2", "s5").get(0));
 		assertEquals(null, innerWriter.query("root.car.d2", "s5").get(1));
+		try {
```
Comment (kr11) [inline src/test/java/cn/edu/tsinghua/tsfile/timeseries/write/ReadPageInMemTest.java]: It's needless here. Messages of thrown exception will be shown and this test case won't be passed, even if you don't catch this exception explicitly.

File: src/test/java/cn/edu/tsinghua/tsfile/timeseries/read/QueryEnginePerf.java (hunk 1 above)
Comment (liukun4515) [reply src/test/java/cn/edu/tsinghua/tsfile/timeseries/read/QueryEnginePerf.java]: move `in.close` after the endTime

File: src/test/java/cn/edu/tsinghua/tsfile/timeseries/read/ReadPerf.java (hunk 2 above)
Comment (liukun4515) [reply src/test/java/cn/edu/tsinghua/tsfile/timeseries/read/ReadPerf.java]: move in.close after the endTime

File: src/test/java/cn/edu/tsinghua/tsfile/timeseries/write/ReadPageInMemTest.java (hunk 3 above)
Comment (liukun4515) [reply src/test/java/cn/edu/tsinghua/tsfile/timeseries/write/ReadPageInMemTest.java]: modified

**Pull request merged**